dasharo-compatibility/reset-to-defaults.robot

```diff
@@ -261,13 +264,14 @@ RTD014.001 F9 resets Memory SPD Profile to JEDEC
     Skip If    not ${MEMORY_PROFILE_SUPPORT}
     Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    RTD014.001 not supported
     Power On
-    Enter Setup Menu Tianocore
-    Enter Dasharo System Features Submenu    Memory Configuration
-    Refresh Serial Screen In BIOS Editable Settings Menu
-    Change To Next Option In Setting    Memory SPD Profile
+    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
+    ${dasharo_menu}=    Enter Dasharo System Features    ${setup_menu}
+    ${memory_menu}=    Enter Dasharo Submenu    ${dasharo_menu}    Memory Configuration
+    Set Option State    ${memory_menu}    Memory SPD Profile    XMP#1 (predefined extreme memory profile)
     Reset To Defaults Tianocore
-    Press Key N Times    1    ${F10}
-    Write Bare Into Terminal    y
-    Read From Terminal Until    ESC to exit
-    ${value}=    Get Option Value    Memory SPD Profile
-    Should Start With    ${value}    <JEDEC
+    Save Changes And Reset    2    4
+    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
+    ${dasharo_menu}=    Enter Dasharo System Features    ${setup_menu}
+    ${memory_menu}=    Enter Dasharo Submenu    ${dasharo_menu}    Memory Configuration
+    ${state}=    Get Option State    ${memory_menu}    Memory SPD Profile
```

Comment:
Keyword `Get Option Value` failed to handle `Memory SPD Profile`.
This option takes two lines, maybe this is a clue. I am attaching the logs.
[log-qemu.zip](https://github.com/Dasharo/open-source-firmware-validation/files/13031088/log-qemu.zip)

![image](https://github.com/Dasharo/open-source-firmware-validation/assets/14311655/cfe5a92e-1a24-4342-9365-23b8ee4a5456)

Reply:
Yes, the problem is that this option spans into more than one line.

Reply:
@Psotas Somehow addresed here: https://github.com/Dasharo/open-source-firmware-validation/pull/55/commits/417898d09e8fb0562657e7d7b96b432cafe1ede1